index.html

```diff
@@ -1,117 +1,283 @@
+                            </li>
+                            <li>
+                              Hacking commences at 10.45 and finishes at 20:00.
+                            </li>                
+                            <li>
+                              You are free to use any tools or libraries available to you to create your product. You can start with any pre-existing code or content that you like and you are free to use third-party assets, as long as you let the judges know what you created yourself and what you didn't. <span class="emphasis">Failure to do so could risk disqualification.</span>
+                            </li>
+                            <li>
+                              It's your responsibilty to make sure that you have the right to use third-party assets (for example, that they are public domain or available under an appropriate license).
+                            </li>
+                            <li>
+                              Be kind and considerate to your fellow hackers and our volunteers. We're all here to have fun! By participating in the hackathon, you agree to abide by <a href="https://hackcodeofconduct.org/" target="_blank">this Code of Conduct</a>.
+                            </li>
+                          </ul>
+                        </div>  
+                        </li>                   
+ 
+                     </ul>                                            
+                </div>
+     </section>
+     <!-- RULES  SECTION END-->
+      <section id="contact-sec"  >
+        <div class="overlay">           
+          <div class="container basic-set">
+            <div class="row " >
+              <div class="col-md-4 col-sm-4" >
+                <h3>OUR LOCATION</h3>
+                <br />
+                <h4>William Gates Building, 15 J.J. Thomson Ave, Cambridge CB3 0FD</h4>
+                <h4>Cambridge, Cambridgeshire</h4>
+                <h4>United Kingdom</h4>
+              <br />
+               <strong>E-mail :</strong> team@hackersatcambridge.com
+                 <br /><br />
+                </div>
+                <div class="col-md-4 col-sm-4">
+                   <h3>WE ARE SOCIAL</h3>
+                    <div class="social">
+                      <a href="https://www.facebook.com/hackersatcambridge/" style="background-color:#3b5998;"><i class="fa fa-facebook-square " ></i>  Facebook </a>
+                      <a href="https://twitter.com/hackersatcam" style="background-color:#55acee;"><i class="fa fa-twitter-square " ></i>  Twitter </a>
+                    </div>
+                   <h6> Drop us a message at any point, regarding the hackathon of just general Hackers at Cambridge matters! </h6>
+             
+              
+                </div>
+               
+               <div class="col-md-4 col-sm-4 subscribe"  >
+                   
+                   <h2 >REGISTER FOR CHARITECH HERE</h2>
```

Comment:
I'm a big fan of semantic HTML, where our content reflects meaning. Having all caps text in HTML is OK, but it has a different meaning to displaying them in all caps as a _design choice_. Namely:

- Search engines will display this in all caps in their site descriptions
- Screen readers for visually impaired users may interpret this different

Would it be better to store this as `<h2>Register for Charitech Here</h2>` and then target it via a CSS rule like `text-appearance: uppercase;`? This would allow us to separate meaning from style, and we wouldn't suffer from the pitfalls mentioned above.